pkgs/by-name/fa/fastqc/package.nix

```
@@ -0,0 +1,77 @@
+{ lib,
+  stdenv,
+  fetchzip,
+  jre,
+  perl,
+  makeWrapper,
+  imagemagick,
+  makeDesktopItem,
+  copyDesktopItems,
+  desktopToDarwinBundle
+}:
+
+stdenv.mkDerivation rec {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```
We prefer to use the `finalAttrs` pattern instead of `rec` to make override easy.